index.html

```diff
@@ -537,26 +565,5 @@ <h2 id="x2.link-type-preload">Link type "<code>preload</code>"</h2>
 </html>
 
 <!-- spec references. preserve before running tidy! -->
-[link]: https://html.spec.whatwg.org/multipage/semantics.html#the-link-element
-[external resource link]: https://html.spec.whatwg.org/multipage/semantics.html#external-resource-link
-[inserted into a document]: https://html.spec.whatwg.org/multipage/infrastructure.html#insert-an-element-into-a-document
-[in a document tree]: https://dom.spec.whatwg.org/#in-a-document-tree
-[Content-Type metadata]: https://html.spec.whatwg.org/multipage/infrastructure.html#content-type
-[valid media query list]: https://html.spec.whatwg.org/multipage/infrastructure.html#valid-media-query-list
-[match the environment]: https://html.spec.whatwg.org/multipage/infrastructure.html#matches-the-environment
-[resolve]: https://html.spec.whatwg.org/multipage/infrastructure.html#resolve-a-url
-[url]: https://html.spec.whatwg.org/multipage/infrastructure.html#url
-[request destination]: https://fetch.spec.whatwg.org/#concept-request-destination
-[request type]: https://fetch.spec.whatwg.org/#concept-request-type
-[request initiator]: https://fetch.spec.whatwg.org/#concept-request-initiator
-[crossorigin]: https://html.spec.whatwg.org/multipage/semantics.html#attr-link-crossorigin
-[origin]: https://html.spec.whatwg.org/multipage/browsers.html#origin-2
-[node document]: https://dom.spec.whatwg.org/#concept-node-document
-[delay the load event]: https://html.spec.whatwg.org/multipage/syntax.html#delay-the-load-event
-[queue a task]: https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-task
-[fire an event]: https://dom.spec.whatwg.org/#concept-event-fire
 [feature-detect]: https://gist.github.com/igrigorik/a02f2359f3bc50ca7a9c
```

Comment:
We should probably link these directly via regular a anchors and drop this section entirely?